src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
```

Comment:
The 'split' function is used correctly to separate the CSS declarations. However, ensure that the input string is properly formatted and does not have trailing semicolons or empty declarations that could result in an empty string in the array.